src/routes/requestRoutes.js

```diff
@@ -9,15 +9,17 @@ import roles from '../middlewares/rolesMiddlewares';
 const app = express.Router();
 
 const { checkExistingTrip, checkLineManager, checkManagerId } = reqMidd;
-const { changeRequestStatus, getManagerRequests } = requestController;
+const { changeRequestStatus, getManagerRequests, search } = requestController;
 const { checkManager, checkRequester } = roles;
-const { checkToken } = userMidd;
-const { tripValidation } = valid;
+const { checkToken, verifyToken } = userMidd;
```

Comment:
good work but these two functions are almost the same. please get rid of one to avoid duplication